staging/src/k8s.io/apiserver/plugin/pkg/audit/truncate/truncate.go

```diff
@@ -129,7 +129,7 @@ func truncate(e *auditinternal.Event) *auditinternal.Event {
 
 func (b *backend) Run(stopCh <-chan struct{}) error {
 	// Nothing to do here
-	return nil
+	return b.delegateBackend.Run(stopCh)
 }
 
 func (b *backend) Shutdown() {
```

Comment:
Should also call b.delegateBackend.Shutdown() here.

Reply:
Done.